src/contexts/Auth/AuthProvider.tsx

```diff
@@ -74,7 +86,17 @@ export const AuthProvider = ({ children }: { children: JSX.Element }) => {
     const logout = async () => {};
 
     return (
-        <AuthContext.Provider value={{ user, login, register, registerCompany, logout }}>
+        <AuthContext.Provider
+            value={{
+                user,
+                login,
+                register,
+                registerCompany,
+                logout,
+                isLogin,
```

Comment:
isLogin verifica se o user tá logado?
Se for, não acha que isLogged seria mais semântico?

Reply:
o isLogin é para verificar se o componente de formulária será o de login ou de cadastro, devo manter o isLogin ou uso outro nome ?

Reply:
Nesse caso o  isLogin funciona bem. Uma outra sugestão (não precisa aplicar, é mais sugestivo mesmo) seria ter um _type_, ex: FormType = 'register' | 'login', e aí ele setaria um dos dois de acordo com a necessidade (ficaria mais legível para uma pessoa que caísse de paraquedas)